package.json

```diff
@@ -10,7 +10,7 @@
     "bootstrap-validator": "^0.11.9",
     "bourbon": "^4.3.4",
     "coffee-script": "^1.11.1",
-    "hel-bootstrap-3": "git+https://github.com/terotic/hel-bootstrap-3.git",
+    "hel-bootstrap-3": "^1.0.0-beta.0",
```

Comment:
`^` should make sure that compatible latest will be installed? 
As we are still in beta some breaking changes are still possible. Maybe change to 1.x or something similar when hel-bootstrap-3 is tested in a few applications?